README.md

```diff
@@ -2,50 +2,184 @@
 
 > Run Lighthouse in CI using Github Actions.
 
-<img align="center" width="998" alt="Lighthouse CI Action" src="https://user-images.githubusercontent.com/158189/65678706-1a063580-e054-11e9-95dc-a1a9fe13bc6b.png">
+<img align="center" width="998" alt="Lighthouse CI Action" src="https://user-images.githubusercontent.com/6392995/68525187-e8ec6800-0283-11ea-8e9e-195802366522.png">
 
-Audit many URLs using [Lighthouse](https://developers.google.com/web/tools/lighthouse),
-and test performance budget as a part of your CI pipeline. This [Github Action](https://github.com/features/actions) makes running Lighthouse in CI easy, free, and without dependencies.
+Audit URLs using [Lighthouse](https://developers.google.com/web/tools/lighthouse),
+and monitor performance with [Lighthouse CI](https://github.com/GoogleChrome/lighthouse-ci).
 
 **Features**:
 
 - ✅ Audit URLs using Lighthouse
-- 🎯 Test performance budget as a part of the build
+- 🎯 Test performance with LHCI assertions
```

Comment:
Maybe Lighthouse CI for better understanding or LHCI acronym makes sense initially? I think many will not be familiar with LHCI, at least initially.